typings/lxml/_types.pyi

```diff
@@ -23,6 +25,9 @@ _TagName: TypeAlias = str
 
 _TagSelector: TypeAlias = _TagName | Callable[..., _Element]
 
+# String argument also support QName in various places
+_TextArg: TypeAlias = str | bytes | QName
```

Comment:
I'm not understanding this comment

Reply:
That comment just came along for the ride from `lxml-stubs`. But I believe the idea is a qualified XML Schema name. So an argument name like `r:rId` that has a namespace ("r:") as well as a name ("rId").

I doubt it will ever come into play for us.